backend/src/test/kotlin/fr/gouv/cacem/monitorenv/infrastructure/api/endpoints/publicapi/ApiMissionsITests.kt

```diff
@@ -290,6 +292,20 @@ class ApiMissionsITests {
         Mockito.verify(deleteMission).execute(20)
     }
 
+    @Test
+    fun `canDelete() should check if a mission can be deleted`() {
+        val missionId = 42
+        val source = MissionSourceEnum.MONITORFISH
+        val canDelete = true
+
+        given(canDeleteMission.execute(missionId = missionId, source = source))
+            .willReturn(canDelete)
```

Comment:
Un autre test pour montrer la réponse en cas d'erreur ?

```suggestion
            .willThrow(exception)
```